pyproject.toml

```diff
@@ -0,0 +1,7 @@
+[tool.black]
+line-length = 120
+exclude = '''
+/(
+  scripts/gen-docs-index
```

Comment:
Is this exclusion necessary? I see that it it existed in the old `pre-commit-config.yaml` config file, but I don't see this file locally.

Reply:
> Is this exclusion necessary? I see that it it existed in the old `pre-commit-config.yaml` config file, but I don't see this file locally.

Thank you! Fixed